web/static/js/study-responses.js

```diff
@@ -110,41 +172,14 @@ $('.star-checkbox').change(function () {
     $(this).labels().children('.icon-star').toggleClass('icon-hidden');
 });
 
-function updateInfoBox(index) {
-    // Select table rows of response details table.
-    const rows = document
-        .querySelector(`#response-summary-${index}`)
-        .querySelectorAll('table tbody tr');
-
-    // construct parent ID
-    const parentName = rows[13].children[1].textContent
-        .toLowerCase()
-        .split(" ")
-        .map(el => ([...el]
-            .filter(v => /[a-z\-]/.test(v)))
-            .join(""))
-        .join("-");
-    const parentId = `${rows[12].children[1].textContent}-${parentName || "anonymous"}`;
-    const parentUUID = rows[11].children[1].textContent
-
-    // Recipient ID for sending message URL
-    const url = new URL(document.querySelector('.contact-family').href);
-    url.searchParams.set("recipient", parentUUID);
-
-    // Short ID for child
-    document.querySelector('.short-child-id').textContent =
-        rows[15].children[1].textContent;
-
-    // Create date for response with formatted date
-    document.querySelector('.response-date').textContent = moment.utc(
-        rows[2].children[1].textContent
-    ).format('M/D/YYYY h:mm A z');
+// On dropdown cell change, update their td's sort and filter values.
+resp_table.on('change', '.dropdown-cell', function () {
+    const td = this.parentElement;
+    const text = this.options[this.selectedIndex].text
```

Comment:
@okaycj we decided that this change to dataset values needs to be integrated with the ["change" event callback that triggers the POST request](https://github.com/lookit/lookit-api/blob/3acdb72bfeb57ebd41c2e77e72ea7a6c73a922d2/web/static/js/study-responses.js#L161-L205), right? If so I will wait to review this.

Reply:
Thanks!  Updated in commit [954fa55](https://github.com/lookit/lookit-api/pull/1527/commits/954fa55bfac3983c350c530ffffa43d5bc1c32ca).